msal/build.gradle

```diff
@@ -105,8 +105,8 @@ task sourcesJar(type: Jar) {
 }
 
 dependencies {
-    implementation "com.android.support:appcompat-v7:$rootProject.ext.supportLibraryVersion"
-    implementation "com.android.support:customtabs:$rootProject.ext.supportLibraryVersion"
+    implementation "androidx.appcompat:appcompat:1.0.0"
```

Comment:
If we're going to define this version here, I'd like to see the constant defined in [`versions.gradle`](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/702/files#diff-25205fe6f2e0d330abcbc45df6b96797R14) either removed or updated and dereferenced here.

Reply:
I updated .gradle in every project (ad-accounts, common, msal, adal) and make them invoke the same versions.gradle file.